### ISSUE TITLE
Can't start, mysql errors

### ISSUE THREAD

**Issue by meiremans** [mativiscusso/ian-ecommerce#17]
so i followed the instructions.

I created manually "ian_commerce" database. (I guess you should add this to the instructions too)
When running yarn start, i get:
[run:server] QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'country_translation' already exists

every time I retry it's a variation of this error. (other table).

How can I solve this? 

thank you

**Comment by darkcherrycreative** [comment]
I did a fresh install of vendure using the command line tool. For me  the issue was that the vendure version here was setup for myself instead of mariadb. I ran though the installer, changed the config file to suit this setup (ports etc) and it can be used fine.  

That said, my test orders never make it into the admin panel in vendure, they get stuck at "adding items to cart" even though I've completed the checkout process.

**Comment by meiremans** [comment]
yes indeed, with my other vendure instance it loads, but almost everything gives an error.
